.circleci/config.yml

```diff
@@ -86,10 +87,22 @@ workflows:
       - orb-tools/dev-promote-prod-from-commit-subject:
           orb-name: instana/pipeline-feedback
           context: orb-publishing
+          checkout: false # We want to control the checkout because we want to inject the version in the scripts
           add-pr-comment: false
           fail-if-semver-not-indicated: true
           publish-version-tag: true
           ssh-fingerprints: 16:0b:f1:b0:af:47:67:d5:69:49:a4:d8:96:a8:42:8b
+          pre-steps:
+            - checkout
+            - run:
+                command: |
+                  CURRENT_VERSION=$(circleci orb info instana/pipeline-feedback | grep 'Latest' | awk '{ print $2 }' | awk -F '@' '{ print $2}' )
+                  SEMVER_INCREMENT=$(echo "${COMMIT_SUBJECT}" | sed -En 's/.*\[semver:(major|minor|patch|skip)\].*/\1/p')
+                  if [ "${SEMVER_INCREMENT}" = 'major' ]; then NEXT_VERSION=$(echo "${CURRENT_VERSION}" | awk 'BEGIN{FS=OFS="."} {$1+=1} 1'); fi
+                  if [ "${SEMVER_INCREMENT}" = 'minor' ]; then NEXT_VERSION=$(echo "${CURRENT_VERSION}" | awk 'BEGIN{FS=OFS="."} {$2+=1} 1'); fi
+                  if [ "${SEMVER_INCREMENT}" = 'patch' ]; then NEXT_VERSION=$(echo "${CURRENT_VERSION}" | awk 'BEGIN{FS=OFS="."} {$3+=1} 1'); fi
+
+                  sed -i '' -e "s/version='dev'.*/version='${NEXT_VERSION}'/" src/scripts/create_release.sh
```

Comment:
Good idea to do modify the whole script file and replace the version rather than have a separate version file